main.cpp

```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
+    ignoreUnused(thisIsTrue);
+    ignoreUnused(thisIsFalse);
+    ignoreUnused(isCorrect);
+    ignoreUnused(firstFloat);
+    ignoreUnused(secondFloat);
+    ignoreUnused(thirdFloat);
+    ignoreUnused(firstDouble);
+    ignoreUnused(secondDouble);
+    ignoreUnused(thirdDouble);
 }
 /*
  10 functions
  example:
  */
 bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
+    { 
     ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
-/*
- 1)
- */
-
-/*
- 2)
- */
-
-/*
- 3)
- */
+    } 
+
+//Function 1
+int yearsWorking(int job1 = 4, int job2 = 10)
+    { 
+    ignoreUnused(job1, job2);
+    return {}; //function returns int.
+    }
+
+//Function 2
+void paintCar(int color, double howMuchPaint)
+    { 
+    ignoreUnused(color, howMuchPaint); // Do I need to include this or just remove this because it returns void?
+    }
+
+//Function 3
+bool careForDog(int hasWater = 1, int hasFood = 0, int hasWalk = 0)
+    { 
+    ignoreUnused(hasWater, hasFood, hasWalk);
+    return {}; //function returns bool.
+    }
+
+//Function 4
+double eatDinner(double numOfBurgers = 1.00000, double numOfFries = 192.0005, int numOfDrink = 5)
+{ 
+    ignoreUnused(numOfBurgers, numOfFries, numOfDrink);
+    return {}; //function returns double.
+}
 
-/*
- 4)
- */
+//Function 5
+void createAccounts(int numOfUsers, float howMuchMoney)
+{
+    ignoreUnused(numOfUsers, howMuchMoney);
+}
 
-/*
- 5)
- */
+//Function 6
+bool backgroundCheck(int numOfCrimes = 0, int numOfTickets = 1)
+{ 
+    ignoreUnused(numOfCrimes, numOfTickets);
+    return {}; //function returns bool.
+}
 
-/*
- 6)
- */
+//Function 7
+bool invest(float returnOnInvestment)
+{ 
+    ignoreUnused(returnOnInvestment);
+    return {}; //function returns bool.
+}
 
-/*
- 7)
- */
+//Function 8
+double record(float lengthOfRecording, float sizeOfRecording, float fileFormat)
+{ 
+    ignoreUnused(lengthOfRecording, sizeOfRecording, fileFormat);
+    return {}; //function returns int.
+}
 
-/*
- 8)
- */
+//Function 9
+bool playSound(int howManyTimes, double howLong)
+{ 
+    ignoreUnused(howManyTimes, howLong);
+    return {}; //function returns bool.
+}
 
-/*
- 9)
- */
+//Function 10
+double doHomework(int amountCompleted, float timeInvested, double finalScore)
+{ 
+    ignoreUnused(amountCompleted, timeInvested, finalScore);
+    return {}; //function returns int.
+}
 
-/*
- 10)
- */
 
 int main()
 {
-    //example of calling that function
-    rentACar(6, 2); 
+    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
+    auto carRented = rentACar(6, 2); 
     
     //1)
+
+    auto workedForYears = yearsWorking(); //already initiailized
     
     //2)
     
+    //Returned Void
+    
```

Comment:
You still need to call them even if they don't return a value.
- [ ] add the function you didn't call.